src/library_browser.js

```diff
@@ -8,8 +8,7 @@
 // Utilities for browser environments
 var LibraryBrowser = {
   $Browser__deps: ['emscripten_set_main_loop', 'emscripten_set_main_loop_timing'],
-  $Browser__postset: 'Module["requestFullScreen"] = function Module_requestFullScreen(lockPointer, resizeCanvas, vrDevice) { err("Module.requestFullScreen is deprecated. Please call Module.requestFullscreen instead."); Module["requestFullScreen"] = Module["requestFullscreen"]; Browser.requestFullScreen(lockPointer, resizeCanvas, vrDevice) };\n' + // exports
-                     'Module["requestFullscreen"] = function Module_requestFullscreen(lockPointer, resizeCanvas, vrDevice) { Browser.requestFullscreen(lockPointer, resizeCanvas, vrDevice) };\n' + // exports
+  $Browser__postset: 'Module["requestFullscreen"] = function Module_requestFullscreen(lockPointer, resizeCanvas, vrDevice) { Browser.requestFullscreen(lockPointer, resizeCanvas, vrDevice) };\n' + // exports
```

Comment:
Please show a good error message in `ASSERTION` builds, something like
```javascript
#if ASSERTIONS
  Module["requestFullScreen"] = Browser.requestFullScreen;

[...]

  requestFullScreen: function() {
    abort("Module.requestFullScreen has been replaced by Module.requestFullscreen (without a capital S)")
  },
#endif
```
This helps people understand and fix breaking changes much faster.

Reply:
Oh, and also please mention it in the Changelog.